### ISSUE TITLE
Improve error handling for sentry-expo

### ISSUE THREAD

**Issue by brentvatne** [expo/expo#282]
```
Running postPublish hook: sentry-expo/upload-sourcemaps
10:00:20 AM
Error: Process exited with non-zero code: 1
   at ChildProcess.child.on (/Users/mharpalani/ag-react-native/node_modules/@expo/spawn-async/build/spawnAsync.js:39:21)
```

This is not useful!

- [ ] wrong node version? or just transpile
- [ ] no sentry-cli globally?
- [ ] config bad?

**Comment by chownation** [comment]
Is this a related error?
```
[exp] Running postPublish hook: sentry-expo/upload-sourcemaps
[exp] Error: spawn sentry-cli ENOENT
    at exports._errnoException (util.js:1050:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:367:16)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

**Comment by brentvatne** [comment]
@chownation - yes, make sure you're configuring it properly -- right now the error isn't so useful.

1) you need to have sentry-cli installed globally
2) you need to have node >= 7.6 because the script requires async/await
3) if those are both satisfied, make sure that you copied the credentials correctly from sentry. you can do this by [supplying them to sentry-cli manually in ENV vars](https://github.com/expo/sentry-expo/blob/de1ce13e912485b439435dfa751307f656efd577/upload-sourcemaps.js#L37-L39) and then running some command from sentry-cli.

**Comment by ide** [comment]
Instead of printing the async error thrown by spawnAsync, perhaps we should print `error.stderr` (spawnAsync adds this property to error objects).

**Comment by brentvatne** [comment]
thanks @ide! fixed this and deployed sentry-expo@1.2.0
